sessions/test_cookies.py

```diff
@@ -713,7 +712,7 @@ class StickyCookieOptions(tester.TempestaTest):
     @dmesg.unlimited_rate_on_tempesta_node
     def check_cannot_start_impl(self, msg):
         self.oops_ignore = ["WARNING", "ERROR"]
-        with self.assertRaises(CmdError, msg=""):
+        with self.assertRaises(error.BaseCmdException, msg=""):
```

Comment:
I meant we should expect `ProcessBadExitStatusException` when we use assertRaises. Please fix all the tests

Reply:
updated